### PR TITLE
Fixed problem that the celery does not start automatically.

### PR DESCRIPTION
**Pull request by piwikjapan**
Fixed problem that the celery does not start automatically.

### CODE REVIEW

File: vagrant/bootstrap.sh [hunk 1]
```diff
@@ -116,6 +116,7 @@ mkdir -p /var/{lib,log,run}/celery
 chown $RUN_AS_USER /var/{lib,log,run}/celery
 cp "${VAGRANT_PATH}"/celery.service /etc/systemd/system/
 cp "${VAGRANT_PATH}"/celery.conf /etc/
+cp "${VAGRANT_PATH}"/tmpfiles.d/celery.conf /usr/lib/tmpfiles.d/
```
Comment (piwikjapan) [inline vagrant/bootstrap.sh]: / var/run/celery disappears when ubuntu starts up. And some functions of timesketch will not work. So, remake it.

File: vagrant/celery.service [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
-After=network.target
+After=network.target redis.service
 
 [Service]
 Type=forking
 User=vagrant
 Group=vagrant
 EnvironmentFile=-/etc/celery.conf
 WorkingDirectory=/var/lib/celery
+ExecStartPre=/bin/sleep 30
```
Comment (berggren) [inline vagrant/celery.service]: This seem to be a bit fragile. Is this to wait for Redis to start up?

File: vagrant/bootstrap.sh (hunk 1 above)
Comment (berggren) [reply vagrant/bootstrap.sh]: ack, good catch.

**Comment by berggren** [comment]
ping @piwikjapan can you take a look at the comment?

File: vagrant/celery.service (hunk 2 above)
Comment (piwikjapan) [reply vagrant/celery.service]: Yes, I am looking for a nice way. It was the result of my Dell E4310.
Do you have any good idea?

**Comment by piwikjapan** [comment]
ping  @berggren Yes, I commented on 4 Mar.

File: vagrant/celery.service (hunk 2 above)
Comment (berggren) [reply vagrant/celery.service]: Sorry for the delay, but I think we can drop the sleep here seeing we adding the wait for redis.service. WDYT?

Comment (piwikjapan) [reply vagrant/celery.service]: I leave this to you.
There is no problem if you can successfully start the timesketch.

**Comment by piwikjapan** [comment]
We were on vacation set by the Japanese government. Please see inline comment.

**Comment by berggren** [comment]
Thanks for contributing!

**Pull request merged**